bootstrap-joining-demo/aws-api-ecs/cfn-templates/ecs-bootstrap-demo-app-infrastructure.yaml

```diff
@@ -1,6 +1,6 @@
 AWSTemplateFormatVersion: '2010-09-09'
 Resources:
-  ECRRepository:
+  Repository:
```

Comment:
was the previous value invalid?

Reply:
Nope, it was fine. Only reason for the change was for consistency with what's done in most of the application template - e.g. the service resource (https://github.com/akka/akka-management/pull/182/files/dafd6d042811fc43c423825110c337bcad8b5669#diff-f41a72438d909aa2145be9a2a27c3a46R106) is just labeled `Service` (not `ECSService`).

In explaining this I realise I could/should have also changed `CloudWatchLogGroup` (https://github.com/akka/akka-management/pull/182/files/dafd6d042811fc43c423825110c337bcad8b5669#diff-f41a72438d909aa2145be9a2a27c3a46R10) to just `LogGroup`.

Reply:
I pushed s/CloudWatchLogGroup/LogGroup/ too for consistency.